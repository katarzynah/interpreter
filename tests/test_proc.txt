program f;

var x, y: int;

procedure add(x, y: int;);
var z: int;
begin
    x := x + 1;
    print(x);
    y := y + 1;
    print(y);
    z := x + y;
    print(z);
end;

begin
    x := 5;
    y := 10;
    print(x);
    print(y);
    add(x, y,);
    print(x);
    print(y);
end.