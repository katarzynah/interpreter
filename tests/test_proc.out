5
10
6
11
17
5
10
